### PR TITLE
Create amule-subdomain.conf.sample

### PR DESCRIPTION
**Pull request by wswv**
Initial release.

the reference docker-compose.yaml file is below:

```
version: '2'
services:
  amule:
    image: tchabaud/amule
    ports:
      - "4711:4711"7#web
      - "4662:4662"#amule used, need to expose to WAN
      - "4672:4672/udp"#amule used, need to expose to WAN
    environment:
      - PUID=1000
      - GUID=1000
    volumes:
      - ./conf:/home/amule/.aMule
      - ./amule/data:/incoming
      - ./amule/tmp:/temp
networks:
  default:
    external: 
      name: SWAG
```

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

**Review by nemchik — CHANGES_REQUESTED** [review]
Thanks for getting this started, but we need the date comment on the first line, and you will need to read the rest of the comments in the file and follow the template instructions in order for this to be merged.

File: amule.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    # location ~ (/<container_name>)?/api {
+    #     include /config/nginx/proxy.conf;
+    #     resolver 127.0.0.11 valid=30s;
+    #     set $upstream_app <container_name>;
+    #     set $upstream_port <port_number>;
+    #     set $upstream_proto <http or https>;
+    #     proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+    #
+    #     # REMOVE THIS LINE BEFORE SUBMITTING: Additional proxy settings such as headers go below this line, leave the blank line above.
+    # }
```
Comment (Roxedus) [inline amule.subdomain.conf.sample:53]: Do what it says

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: amule.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        # enable the next two lines for ldap auth
+        #auth_request /auth;
+        #error_page 401 =200 /ldaplogin;
+
+        # enable for Authelia
+        #include /config/nginx/authelia-location.conf;
+
+        include /config/nginx/proxy.conf;
+        resolver 127.0.0.11 valid=30s;
```
Comment (Roxedus) [inline amule.subdomain.conf.sample:33]: Please update this to reflect the changes brought by #330

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Comment by Roxedus** [comment]
Closed as requested changes has not been implemented.

**Pull request closed without merging**